portal/templates/redesign/teach_new/class_new.html

```diff
@@ -4,17 +4,36 @@
 
 {% block css %}
 {{ block.super }}
+<link href='{% static "portal/css/jquery-ui.css" %}' rel="stylesheet">
 {% endblock css %}
 
 {% block scripts %}
 {{ block.super }}
+<script type="text/javascript" src="{% static 'portal/js/teach_class.js' %}"></script>
+<script type="text/javascript" src="{% static 'portal/js/common.js' %}"></script>
+<script type="text/javascript" src="{% static 'portal/js/lib/jquery-ui.js' %}"></script>
+<script>
+    var CLASS_CODE = "{{ class.access_code }}";
```

Comment:
I take it that this is feeding preexisting JS?  If this is for new stuff, I'd recommend other ways of getting these data into the script.